### PR TITLE
Add dcos k8s beer demo

### PR DESCRIPTION
**Pull request by rimusz**

### CODE REVIEW

**Comment by rimusz** [comment]
cc @joerg84

File: README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@ Overview of demos in 1.10:
 
 * [Spring Boot Deployments](springboot/1.10/README.md#springboot-demo)
 * [Apache Flink Stream Processing](flink/1.10#fast-data-financial-transaction-processing-with-apache-flink)
+* [DC/OS + Kubernetes + Helm + Cloudflare Warp - beer demo](dcos-k8s-beer-demo/1.10#the-dcos--kubernetes--helm---cool-beer-demo)
```
Comment (joerg84) [inline README.md]: Change name here as well

File: dcos-k8s-beer-demo/1.10/README.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+Are you wondering how [Java](http://www.oracle.com/technetwork/java/index.html), [Spring Boot](https://projects.spring.io/spring-boot/), [MySQL](https://www.mysql.com), [Neo4j](https://neo4j.com), [Apache Zeppelin](https://zeppelin.apache.org/), [Apache Spark](https://spark.apache.org/), [Elasticsearch](https://www.elastic.co),  [Apache Mesos](https://mesos.apache.org/), [DC/OS](https://dcos.io), [Kubernetes](https://kubernetes.io/) and [Helm](https://helm.sh) can all fit in one demo? Well, we'll show you! This is a cool demo, so grab your favourite beer and enjoy. 🍺
+
+**Note:** It is based on [dcos-beer-demo](https://github.com/unterstein/dcos-beer-demo), but `beer-service` is running on Kubernetes and is exposed to Internet with [Cloudflare Warp](https://warp.cloudflare.com/docs/kubernetes/).
+
+So that means all the backend is running on DC/OS and Internet facing services are running on Kubernetes.
+
+## DC/OS cluster with Kubernetes
+
+**Note:** In order to deploy all parts of this demo successfully, I would recommend to spin up a cluster with at least **7** private DC/OS nodes, also you will need Kubernetes package to be installed.
```
Comment (joerg84) [inline dcos-k8s-beer-demo/1.10/README.md]: Add link to dcos install and command to install k8.

Comment (joerg84) [reply dcos-k8s-beer-demo/1.10/README.md]: I see below, reduce this section to DC/OS 1.10 cluster?

File: dcos-k8s-beer-demo/1.10/README.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+helm version
+Client: &version.Version{SemVer:"v2.7.2", GitCommit:"8478fb4fc723885b155c924d1c8c410b7a9444e6", GitTreeState:"clean"}
+Server: &version.Version{SemVer:"v2.7.2", GitCommit:"8478fb4fc723885b155c924d1c8c410b7a9444e6", GitTreeState:"clean"}
+```
+
+## Deploy Backend on DC/OS
+
+Let's deploy Backend using the `dcos` cli:
+```bash
+dcos marathon group add marathon-apps/marathon-configuration.json
```
Comment (joerg84) [inline dcos-k8s-beer-demo/1.10/README.md]: We could replace that with URL once merged (no need to clone example repo)

File: dcos-k8s-beer-demo/1.10/README.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+
+Wait till it gets installed, you can check it's progress in DC/OS Dashboard/Services/beer.
+
+## Deploy Frontend App and Cloudflare Warp on Kubernetes
+
+### Frontend App
+
+To deploy Frontend App run (do not forget to replace there with your `domain_name`):
+```bash
+helm install --name beer --namespace beer helm-charts/beer-service --set ingress.host=beer.mydomain.com
```
Comment (joerg84) [inline dcos-k8s-beer-demo/1.10/README.md]: Same here.

Comment (joerg84) [reply dcos-k8s-beer-demo/1.10/README.md]: Also we only need the ingress in connection with the cloudflare setting, corrrect?

**Comment by joerg84** [comment]
The comments can be fixed in a later PR.

File: dcos-k8s-beer-demo/1.10/marathon-apps/marathon-configuration.json [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+   "apps":[
+      {
+         "id":"/beer/database",
+         "cpus":1,
+         "mem":1024,
+         "instances":1,
+         "container":{
+            "type":"DOCKER",
+            "docker":{
+               "image":"unterstein/dcos-beer-database",
```
Comment (joerg84) [inline dcos-k8s-beer-demo/1.10/marathon-apps/marathon-configuration.json]: Let us move the docker images to the mesosphere account.

**Pull request merged**